packages/preview-middleware/test/unit/base/flp.test.ts

```diff
@@ -336,7 +341,7 @@ describe('FlpSandbox', () => {
 
         test('rta with developerMode=true and plugin', async () => {
             await server.get('/with/plugin.html').expect(200);
-            const response = await server.get('/with/plugin.html.inner.html').expect(200);
+            const response = await server.get('/with/plugin.html.inner.html').expect(302);
```

Comment:
We control exactly what is called as `inner.html` therefore, we should be able to also add the url parameters if required, thus, I would not expect that there is a redirect necessary for the inner html.
Is the test case just wrong i.e. should we test here with parameters or is there an unnecessary redirect now?

Reply:
Are the URL parameters needed at all in case of `inner.html`?

Reply:
Thanks for adjusting the test case